src/librustc_resolve/diagnostics.rs

```diff
@@ -641,7 +641,7 @@ impl<'a> Resolver<'a> {
         add_typo_suggestion(err, suggestion, ident.span);
 
         if macro_kind == MacroKind::Derive &&
-           (ident.as_str() == "Send" || ident.as_str() == "Sync") {
+           (ident.name == sym::Send || ident.name == sym::Sync) {
```

Comment:
This case is hit during only during error reporting, so it may be slow, and in that case we won't need to pre-intern `Send` and `Sync`.
The PR in general can be audited for cases hit only during error reporting, so they don't add *new* entries into the pre-interned list.
(Not sure how much this matters though.)

Reply:
This change is no longer present.